pilot/lib/pilot/tester_importer.rb

```diff
@@ -13,8 +13,10 @@ def import_testers(options)
       tester_manager = Pilot::TesterManager.new
       imported_tester_count = 0
 
+      group_name = options[:group]
+
       CSV.foreach(file, "r") do |row|
-        first_name, last_name, email = row
+        first_name, last_name, email, testing_group = row
```

Comment:
It seems to me that this change will not be compatible with the exporter. This is how the CSV is generated:

```csv << [tester.first_name, tester.last_name, tester.email, tester.devices.count, group_names, #...```